src/main/cpp/alias/Alias.cpp

```diff
@@ -0,0 +1,77 @@
+/*
+ * Alias.cpp
+ *
+ *  Created on: Mar 11, 2017
+ *      Author: cedric
+ */
+
+#include "Alias.h"
+#include "AliasUtil.h"
+
+#include <exception>
+#include <assert.h>
+#include <math.h>
+
+namespace stride {
+namespace alias {
+
+Alias& Alias::CreateDistribution(std::vector<double> probabilities, util::Random& rng)
```

Comment:
Should it be return *(new Alias(alias,prob,rng))? I don't know any other way make a static function return a reference in c++.